File: TerraObserver/TO.FSharp/GodotAbstractions/Bases/IMeshInstance3D.fs
namespace TO.FSharp.GodotAbstractions.Bases

open Godot

/// Copyright (C) 2025 Zhu Xiaohe(aka ZeromaXHe)
/// Author: Zhu XH (ZeromaXHe)
/// Date: 2025-05-28 23:03:28
[<Interface>]
type IMeshInstance3D =
    inherit IGeometryInstance3D
    abstract Mesh: Mesh with get, set

File: TerraObserver/Godot.Abstractions/Bases/IMeshInstance3D.cs
namespace Godot.Abstractions.Bases;

/// Copyright (C) 2025 Zhu Xiaohe(aka ZeromaXHe)
/// Author: Zhu XH
/// Date: 2025-04-17 14:15:17
public interface IMeshInstance3D : IGeometryInstance3D
{
    Mesh? Mesh { get; set; }
}